epochStart/metachain/rewardsV2.go

```diff
@@ -381,7 +381,7 @@ func (rc *rewardsCreatorV2) computeTopUpRewards(totalToDistribute *big.Int, tota
 	}
 
 	// k = c * economics.TotalToDistribute, c = top-up reward factor (constant)
-	k := core.GetPercentageOfValue(totalToDistribute, rc.topUpRewardFactor)
+	k := core.GetIntTrimmedPercentageOfValue(totalToDistribute, rc.topUpRewardFactor)
```

Comment:
This is not yet activated, isn't it? Otherwise, we need a flag for backwards compatibility here.

Reply:
Correct, rewardsv2 is activated with staking v2, which is not yet enabled, so we don't need backwards compatibility implemented.